docs/topics/execute.adoc

```diff
@@ -68,6 +68,49 @@ To override the default _Fernflower_ decompiler, pass the `-Dwindup.decompiler`
 $ __RHAMT_HOME__/bin/rhamt-cli -Dwindup.decompiler=procyon --input __INPUT_ARCHIVE_OR_DIRECTORY__ --output __OUTPUT_REPORT_DIRECTORY__ --target __TARGET_TECHNOLOGY__ --packages __PACKAGE_1__ __PACKAGE_2__
 ----
 
+[discrete]
+[[cli_bash_completion]]
+== {ProductShortName} {CLINameTitle} Bash Completion
+
+The {ProductShortName} {CLIName} includes a set of bash completions, allowing the xref:command_line_arguments[{ProductShortName} Command-line Arguments] to be auto completed by pressing Tab when entering the commands. For instance, entering the following displays a list of available arguments.
+
+[source,options="nowrap",subs="+quotes"]
+----
+$ __RHAMT_HOME__/bin/rhamt-cli [TAB]
+----
+
+[discrete]
+[[bash_completion_temporary]]
+=== Enable Bash Completion
+
+To enable bash completion for the current shell execute the following command. After the prompt returns follow the steps in xref:execute[Run the {CLIName}] as normal.
+
+[source,options="nowrap",subs="+quotes"]
+----
+$ . __RHAMT_HOME__/bash-completion/rhamt-cli
```

Comment:
Looks like this has an extra period at the beginning of the command.

Reply:
The `rhamt-cli` file included isn't executable by default, meaning executing `./bash-completion/rhamt-cli` fails.  The period is another of way of saying to load the file.

To be more explicit I could replace the period with `source`, as it accomplishes the same thing.  What do you think?

Reply:
On second thought I went ahead and adjusted the period to say `source`, keeping the rest of the line the same.

Reply:
Ah, I am familiar with using source - I didn't realize that using a period accomplished the same thing. Agreed to using source to make it more obvious, thanks!